hudi-integ-test/src/test/java/org/apache/hudi/integ/ITTestBase.java

```diff
@@ -132,6 +134,12 @@ public void init() {
     await().atMost(60, SECONDS).until(this::servicesUp);
   }
 
+  private void suppressSparkLogs() {
```

Comment:
this may not have that much impact. given we don't kick off any Spark DAGs from this java class. its all done via commands passes to docker.